view/theme/frio/style.php

```diff
@@ -49,7 +49,7 @@
 $modified         = time();
 
 if (DI::mode()->has(\Friendica\App\Mode::MAINTENANCEDISABLED)) {
-	DI::config()->load('frio');
+	DI::config()->reload('frio');
```

Comment:
No parameter anymore.
```suggestion
	DI::config()->reload();
```